tests/message/unittest_message_definition.py

```diff
@@ -21,7 +21,7 @@
         ("W12345", "Invalid message id 'W12345'"),
     ],
 )
-def test_create_invalid_message_type(msgid, expected):
+def test_create_invalid_message_type(msgid: str, expected: str) -> None:
```

Comment:
As the code owner of ``tests/message``, I approve of this change ! (I think we bothered areveny and DudeNr33 with the typing changes because of codeowner but it should be a one time thing)